### PR TITLE
add documentation about http headers used by the backend

### LINKED ISSUE #384: document relevant http headers we use
- [ ] `Accept-Language`
- [ ] auth token

### PR DESCRIPTION
**Pull request by kitzbergerg**
Add http header documentation.

Resolves #384.

## Basics

- [x] The PR is rebased with current master.
- [x] Details of what you changed are in commit messages.
- [x] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildserver is happy.

## Checklist

- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation
- [ ] I fixed all affected decisions
- [ ] I added unit tests for my code
- [ ] I added code comments, logging, and assertions as appropriate
- [ ] I translated all strings visible to the user
- [ ] I mentioned [every code or binary](/.reuse/dep5) not directly written or done by me in [reuse syntax](https://reuse.software/)
- [ ] I created left-over issues for things that are still to be done
- [ ] Code is conforming to [our Architecture](/doc/architecture)
- [ ] Code is conforming to [our Guidelines](/doc/guidelines)
      (exceptions are documented)
- [ ] Code is consistent to [our Design Decisions](/doc/decisions)

## Review

- [ ] I've tested the code
- [ ] I've read through the whole code
- [ ] Examples are well chosen and understandable

### CODE REVIEW

File: doc/backend/04http_header.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# HTTP Headers
+
+This document explains the HTTP headers used by the backend.
```
Comment (markus2330) [inline doc/backend/04http_header.md:3]: received or sent?

File: doc/backend/04http_header.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+## Authorization
+
+The Authorization header is used for authentication/authorization. The access token should be set here.
```
Comment (markus2330) [inline doc/backend/04http_header.md:7]: Who sets it?

File: doc/backend/04http_header.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# HTTP Headers
+
+This document explains the HTTP headers used by the backend.
+
```
Comment (markus2330) [inline doc/backend/04http_header.md:4]: Link https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html here.

**Review by markus2330 — CHANGES_REQUESTED** [review]
Good start. What about "Accept-Language", is it used? Are there some headers set/consumed by actix? Please write for each if they are sent and/or received.

File: doc/backend/04http_header.md (hunk 3 above)
Comment (kitzbergerg) [reply doc/backend/04http_header.md:4]: I linked mozilla doc instead. The one you linked is stated to be superseded and mozilla has a more specific one.

**Pull request merged**